slmsuite/holography/toolbox/__init__.py

```diff
@@ -967,7 +969,7 @@ def _process_grid(grid):
     return grid
 
 
-import slmsuite.holography.toolbox.phase as toolbox_phase
```

Comment:
Recommend keeping "toolbox_phase" to clarify, but it's currently not used either way.